### PR TITLE
My Finished Submission

### PR DESCRIPTION
**Pull request by Ousman37**
Please review My social media app  site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,211 @@
...
+  <div class="container mt-5 pt-5 d-flex flex-column flex-lg-row justify-content-evenly">
+    <!--! heading -->
+    <div class="text-center text-lg-start mt-0 pt-0 mt-lg-5 pt-lg-5">
+      <a href="/feed.html"><button class="btn btn-primary w-50 my-1 text-light">Back to home </button></a>
+      <h1 class="text-primary fw-bold fs-0">Inbound Media</h1>
+      <p class="w-75 mx-auto fs-4 mx-lg-0">Inbound Media helps you connect and share with the people in your life.</p>
+      
+    </div>
+    <!-- !form card -->
+    <div style="max-width: 28rem; width: 100%">
```
Comment (Fermain) [inline index.html]:
```suggestion
    <div class="my-custom-class">
```

Please do not use inline styles. This is not best practice as per HTML + CSS course.

File: index.html [hunk 2]
```diff
@@ -1,13 +1,211 @@
...
+      <!-- !login form -->
+      <!-- !first was form tag -->
+      <div class="bg-white shadow rounded p-3 input-group-lg">
+      <form>
+        <div class="form-group">
+          <label for="email">Email</label>
+          <input pattern="^[A-Za-z0-9._+-]+@[A-Za-z0-9.-]+\.[a-z]{2,4}"   id="email" required   type="email"  class="form-control"   title="Must be a valid email address"
```
Comment (Fermain) [inline index.html]: Phone inputs will not be valid under this pattern, I suggest you change the placeholder text to remove phone

File: index.html [hunk 3]
```diff
@@ -1,13 +1,211 @@
...
+    <!-- !form card -->
+    <div style="max-width: 28rem; width: 100%">
+      <!-- !login form -->
+      <!-- !first was form tag -->
+      <div class="bg-white shadow rounded p-3 input-group-lg">
+      <form>
+        <div class="form-group">
+          <label for="email">Email</label>
+          <input pattern="^[A-Za-z0-9._+-]+@[A-Za-z0-9.-]+\.[a-z]{2,4}"   id="email" required   type="email"  class="form-control"   title="Must be a valid email address" 
+           placeholder="name@example.com or phone number"/>
```
Comment (Fermain) [inline index.html]:
```suggestion
           placeholder="name@example.com"/>
```

File: index.html [hunk 4]
```diff
@@ -1,13 +1,211 @@
...
+       
+          <div class="form-group">
+              <label for="password" class="form-control">Password</label>
+              <input title=" Must be at least 8 characters" pattern="[a-zA-Z0-9]{8,}"  class="form-control" required type="text" id="password" />
+                 <p class="alert alert-danger"> Must be at least 8 characters</p>
+          </div>
+     
+      
+       
+          <a href="/feed.html"><button class="btn btn-primary w-100 my-3 text-light">Log In</button></a>
```
Comment (Fermain) [inline index.html]:
```suggestion
          <button class="btn btn-primary w-100 my-3 text-light">Log In</button>
```

If you add a form action, you don't need the extra <a> tag. It's also invalid to nest a button inside an anchor tag or vice versa.

File: index.html [hunk 5]
```diff
@@ -1,13 +1,211 @@
...
+      <h1 class="text-primary fw-bold fs-0">Inbound Media</h1>
+      <p class="w-75 mx-auto fs-4 mx-lg-0">Inbound Media helps you connect and share with the people in your life.</p>
+      
+    </div>
+    <!-- !form card -->
+    <div style="max-width: 28rem; width: 100%">
+      <!-- !login form -->
+      <!-- !first was form tag -->
+      <div class="bg-white shadow rounded p-3 input-group-lg">
+      <form>
```
Comment (Fermain) [inline index.html]:
```suggestion
      <form action="/profile.html">
```

This is a requirement in the CA

File: src/scss/styles.scss [hunk 6]
```diff
@@ -1,9 +1,169 @@
...
-    background-color: $blue;
+
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
+body{
+  font-family: "Roboto", sans-serif;
+}
+
+.mt-5 {
+  margin-top: -10rem !important;
```
Comment (Fermain) [inline src/scss/styles.scss]: I really do not recommend you do this. You can create a custom class with this style if you need but this is not a good way to extend bootstrap.

**Review by Fermain — CHANGES_REQUESTED** [review]
This is really impressive. I am not sure how you did this so quickly to be honest.

There are some things to be cleaned up - like inline styles, there shouldn't be any. If you can get these resolved before Sunday this will be a top submission.

File: index.html (hunk 3 above)
Comment (Ousman37) [reply index.html]: change has been updated

File: index.html (hunk 5 above)
Comment (Ousman37) [reply index.html]: Done

File: src/scss/styles.scss (hunk 6 above)
Comment (Ousman37) [reply src/scss/styles.scss]: Okay, thanks you understand. I will keep  that in mind for next time.

**Comment by Ousman37** [comment]
All changing has been updated I hope I am missing something else  I will go through on it tomorrow to double check again.

**Pull request merged**